test/JDBC/expected/sys_database_principals-vu-verify.out

```diff
@@ -0,0 +1,118 @@
+use sys_database_principals_vu_db
+GO
+
+-- create a login which is a member of sysadmin server role
+create login sys_database_principals_vu_login_with_sysadmin with password = '12345678'
+alter server role sysadmin add member sys_database_principals_vu_login_with_sysadmin
+GO
+
+-- change db owner to the new sysadmin login
+alter authorization on database::sys_database_principals_db_different_owner to sys_database_principals_vu_login_with_sysadmin
```

Comment:
Let's make unprivileged login owner of database and connect with that login to get side of dbo.